.github/ISSUE_TEMPLATE/pvc_request.yaml

```diff
@@ -0,0 +1,26 @@
+name: "User support: JupyterHub hardware requirements"
+description: "Template for users of JupyterHub requesting hardware quota upgrade: disk size, CPU/GPU/Memory."
+title: "PROJECT: <name>"
+labels: [kind/onboarding, area/JupyterHub]
+assignees: [first-operator]
+body:
+    - type: input
+      id: moc-name
+      attributes:
+          label: User details
+          description: Your MOC user name
```

Comment:
Let's not mention MOC here. User name is enough (like in the other form file above).

Reply:
Resolved in latest commit.